### ISSUE TITLE
Add convenience API to add multiple objects to a scene at once

### ISSUE THREAD

**Issue by JohnSundell** [JohnSundell/ImagineEngine#63]
It would be nice to have a convenience API that lets the user add multiple actors, blocks or labels to a scene at once. So that you can write code like this:

```swift
let actorA = Actor()
let actorB = Actor()
let actorC = Actor()
scene.add(actorA, actorB, actorC)

let blockA = Block(size: .zero, textureCollectionName: "")
let blockB = Block(size: .zero, textureCollectionName: "")
let blockC = Block(size: .zero, textureCollectionName: "")
scene.add(blockA, blockB, blockC)

let labelA = Label()
let labelB = Label()
let labelC = Label()
scene.add(labelA, labelB, labelC)
```

The convenience API can be added as an extension on `Scene` 👍

**Comment by NSMyself** [comment]
Can i do this one?

**Comment by JohnSundell** [comment]
@NSMyself Of course! Go for it 👍

**Comment by NSMyself** [comment]
This is pretty straightforward to implement, in my opinion:
```
public func add(_ actors: Actor...) {
    actors.forEach {
        add($0)
    }
}

public func add(_ labels: Label...) {
    labels.forEach {
        add($0)
    }
}
    
public func add(_ blocks: Block...) {
    blocks.forEach {
        add($0)
    }
}
```

I'm not fond of having to repeat that `forEach` cycle for every possible class that conforms to the `SceneObject` protocol though. 

In my opinion, a generic `add` method that supported a variable number of objects (all of which that comply with the `SceneObject` protocol) would make sense. Do you agree? Or do you prefer having to extend the Scene class explicitly for each new class?

**Comment by JohnSundell** [comment]
@NSMyself Yeah, or you can do it event simpler actually 😀

```swift
func add(_ actors: Actor...) {
    actors.forEach(add)
}
```

I totally agree that we should try to unify all operations that deal with `SceneObject`s. I started doing that in #52 and #54, and it's an effort we should continue to get rid of as much code duplication as possible 👍 I don't think it's within the scope of this issue though, but rather something we need to do on an ongoing basis going forward.

**Comment by NSMyself** [comment]
Nicely spotted 👍 thanks!

OK, looks good, I'll open a PR then.

**Comment by JohnSundell** [comment]
Fixed by @NSMyself in #77 🎉
